src/js/containers/state/StateContainer.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * StateContainer.jsx
+ * Created by Lizzie Salita 5/1/18
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { isCancel } from 'axios';
+
+import BaseStateProfile from 'models/v2/state/BaseStateProfile';
+import * as StateHelper from 'helpers/stateHelper';
+import * as stateActions from 'redux/actions/state/stateActions';
+
+import StatePage from 'components/state/StatePage';
+
+require('pages/state/statePage.scss');
+
+const propTypes = {
+    params: PropTypes.object,
+    stateProfile: PropTypes.object,
+    setStateOverview: PropTypes.func,
+    setStateFiscalYear: PropTypes.func
+};
+
+export class StateContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loading: true,
+            error: false
+        };
+
+        this.request = null;
+    }
+    componentWillMount() {
```

Comment:
As we move toward React 16, please avoid using [deprecated lifecycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html).

Reply:
Changed to `componentDidMount` and `componentDidUpdate`